fetcher/tar.py

```diff
@@ -99,18 +101,26 @@ def unpack(self, downloadFirst=False):
         update = tarfile.open(name=self.workspacefile, mode='r:*')
```

Comment:
Well, if you also want to keep the tarFetcher, you will need to adapt it to work with streams I think to better accommodate both cases. So it can work with such a boolean, plus creating a stream out of the tar archive the tar fetcher downloads